### PR TITLE
feat: provide guidence for learning open API (#489)

### LINKED ISSUE #489: Provide more guidance for learning/exploring OpenAPI.
Add something like:

A good way to learn more about OpenAPI is to explore the [OpenAPI specification mind map](https://openapi-map.apihandyman.io/) and use our code completion providing [Swagger Plugin for IntelliJ](https://plugins.jetbrains.com/search?search=swagger+Monte).

### PR DESCRIPTION
**Pull request by tkrop**
closes #489.

### CODE REVIEW

File: chapters/general-guidelines.adoc [hunk 1]
```diff
@@ -21,18 +21,23 @@ all apis with `x-api-audience =/= component-internal` (see <<219, API Audience>>
...
+
+The API specification files should be subject to version control using a source
+code management system - best together with the implementing sources. 
+
+You <<192, must / should publish>> the component <<219, external / internal>>
+API specification with the deployment of the implementing service, and, hence,
+make it discoverable for the group via our https://apis.zalando.net/[API Portal
+[internal link]].
+
+*Hint:* A good way to explore *OpenAPI 3.0/2.0* is to navigate through the
```
Comment (maxim-tschumak) [inline chapters/general-guidelines.adoc]: let's also add Swagger online editor (e.g. to check the validity of the document).

Comment (tkrop) [reply chapters/general-guidelines.adoc]: Okay, I add a second sentence like:

If you want explore and validate existing APIs, you may utilize the [Swagger Editor](https://editor.swagger.io/) or our [API Portal](https://apis.zalando.net).

**Comment by whiskeysierra** [comment]
:+1:

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by tkrop** [comment]
:+1:

**Pull request merged**